components/d2l-activity-editor/d2l-activity-quiz-editor/lang/en.js

```diff
@@ -71,6 +71,7 @@ export default {
 	"textIsDisplayedSingularPart2": "The {field} field appears to learners when the quiz is saved.", // Second sentence of alert warning that text fields will be visible, when there is only one field
 	"introMovedToDescription": "Quizzes no longer support Introductions. Existing content in the Introduction field now appears in the Description field.", // Alert warning that existing quiz introduction has been removed and that the text in the introduction has been put into the quiz description
 	"attemptsAllowed": "Attempts Allowed", // Title for Attempts Allowed select in Attempts dialog
+	"quizAttemptsAllowedSummary": "{numAttemptsAllowed, plural, =1 {1 attempt} other {{numAttemptsAllowed} attempts}} allowed", //
```

Comment:
Best not to split up the string like this (see [this PSA](https://d2l.slack.com/archives/C0TF4KSGZ/p1614349569011100))
```suggestion
	"quizAttemptsAllowedSummary": "{numAttemptsAllowed, plural, =1 {1 attempt allowed} other {{numAttemptsAllowed} attempts allowed}}", //
```

Reply:
Oooh interesting, thanks for catching that!